src/lib/components/sidebar/Documents.svelte

```diff
@@ -43,47 +43,57 @@
   }
 </script>
 
-<Flex direction="column">
-  <SidebarItem hover href={searchUrl("")} active={query === ""}>
-    <Infinity16 slot="start" />
-    {$_("documents.allDocuments", {
-      values: { access: "" },
-    })}
+<SidebarGroup name="documents">
+  <SidebarItem slot="title">
+    <File16 slot="start" />
+    Documents
   </SidebarItem>
-
+  <Button
+    slot="action"
+    ghost
+    minW={false}
+    mode="primary"
+    size="small"
+    href="/documents"
+  >
+    <Search16 height="14" width="14" />
+    {$_("common.explore")}
+  </Button>
   <SignedIn>
-    <SidebarItem hover href={searchUrl(mine)} active={query === mine}>
-      <File16 slot="start" />
-      {$_("documents.yourDocuments")}
-    </SidebarItem>
-
```

Comment:
We want to keep this link to "all your documents," in addition to public, private and org docs.